app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,8 +1,17 @@
 class Admin::DashboardController < Admin::AdminController
+
+  STATUS_FILTERS = ['all', 'active', 'completed']
```

Comment:
Here is fine if it's just for this view, which I think it is. Please use the `%w(...)` notation though.